src/material-experimental/mdc-autocomplete/_autocomplete-theme.scss

```diff
@@ -11,7 +11,7 @@
 }
 
 @mixin mat-mdc-autocomplete-typography($config-or-theme) {
-  $config: mat-get-typography-config($config-or-theme);
+  $config: mat-private-typography-normalized-config(mat-get-typography-config($config-or-theme));
```

Comment:
It seems like we're repeating `mat-private-typography-normalized-config(mat-get-typography-config())` a lot. Should we create a helper function for it?

Reply:
I plan to just fold the call to `mat-private-typography-normalized-config` into `mat-get-typography-config`, but for now I'm doing it this way because I think for some of the components it may result in a number of screen diffs that need to be resolved in g3. Once all the components are using it I'll fold it in.